File: android/src/main/java/ru/whalemare/rn/library/Payment.kt
package ru.whalemare.rn.library

import ru.yandex.money.android.sdk.PaymentMethodType

/**
 * @since 2019
 * @author Anton Vlasov - whalemare
 */
data class Payment(
  val amount: Double,
  val currency: String,
  val types: Set<PaymentMethodType>
)

File: ios/RNYandexPayment/Payment.swift
//
//  Payment.swift
//  RNYandexPayment
//
//  Created by Антон Власов on 04/09/2019.
//  Copyright © 2019 whalemare. All rights reserved.
//

import Foundation
import YandexCheckoutPayments
import YandexCheckoutPaymentsApi

struct Payment {
    let amount: Double
    let currency: Currency
    let types: Set<PaymentMethodType>
}
